app/jobs/penn_import_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Imports the TEI files from the github directory configured in +Settings.import.directory.penn+
+class PennImportJob < GithubImportJob
+  private
+
+  def process_file(filename, tei)
+    identifier = "penn_#{filename.sub('.xml', '')}"
+    process_tei(identifier, tei)
+  end
+
+  def import_directory
+    Settings.import.directory.penn
+  end
+
+  def process_tei(identifier, tei)
+    indexer = Traject::Indexer.new('identifier' => identifier, 'exhibit_slug' => Settings.import.slug)
```

Comment:
If we also load `config/traject.rb`, we ought to get the exhibit_slug for "free".